<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HP" failures="1" tests="3" name="com.naveenautomation.tests.GiftCertificatePageTest" time="80.637" errors="0" timestamp="2024-03-18T22:52:02 EDT" skipped="2">
  <testcase classname="com.naveenautomation.tests.GiftCertificatePageTest" name="validateGiftCertificate" time="26.518">
    <skipped/>
  </testcase> <!-- validateGiftCertificate -->
  <system-out/>
  <testcase classname="com.naveenautomation.tests.GiftCertificatePageTest" name="validateGiftCertificate" time="27.655">
    <skipped/>
  </testcase> <!-- validateGiftCertificate -->
  <system-out/>
  <testcase classname="com.naveenautomation.tests.GiftCertificatePageTest" name="validateGiftCertificate" time="26.464">
    <failure message="expected [Thank you for purchasing a goft certificate! Once you have completed your order your gift certificate recipient will be sent an e-mail with details how to redeem their gift certificate.] but found [Thank you for purchasing a gift certificate! Once you have completed your order your gift certificate recipient will be sent an e-mail with details how to redeem their gift certificate.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Thank you for purchasing a goft certificate! Once you have completed your order your gift certificate recipient will be sent an e-mail with details how to redeem their gift certificate.] but found [Thank you for purchasing a gift certificate! Once you have completed your order your gift certificate recipient will be sent an e-mail with details how to redeem their gift certificate.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.naveenautomation.pages.GiftCertificatePage.validateGiftCertificate(GiftCertificatePage.java:132)
at com.naveenautomation.pages.GiftCertificatePage.validateGiftCertificateFlow(GiftCertificatePage.java:155)
at com.naveenautomation.tests.GiftCertificatePageTest.validateGiftCertificate(GiftCertificatePageTest.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- validateGiftCertificate -->
  <system-out/>
</testsuite> <!-- com.naveenautomation.tests.GiftCertificatePageTest -->
